pytorch_lightning/plugins/precision/hpu.py

```diff
@@ -50,6 +50,7 @@ def __init__(
             )
         super().__init__()
         self.precision = precision
-        hmp.convert(
-            opt_level=opt_level, bf16_file_path=bf16_file_path, fp32_file_path=fp32_file_path, isVerbose=verbose
-        )
+        if precision in (16, "bf16"):
```

Comment:
```suggestion
        if precision in (PrecisionType.HALF, PrecisionType.BFLOAT):
```
let's use enum. also, you might need to import it